### ISSUE TITLE
Settings | Options, switching between tabs crashes QGIS

### ISSUE THREAD

**Issue by bellanz80** [qgis/QGIS#39725]
Hi. After installing QGIS 3.16 on windows, I get several crashes and processing plug errors.
Initially I thought they were the plugins of my profile (I have been using QGIS 3.14 since time without any errors, and some plugins).
But no, the error is presented with the clean, new profile.
I did several tests. I had come to the conclusion that the problem was the gray color theme of QGIS (it was the only thing that always changed when creating a new profile).
But in the last tests, it is also presented without changing the color of the theme.

How to reproduce the error:

1) create new profile
2) enter the settings / options menu
3) move between the tabs on the left side
4) in many cases, going through the RENDERING tab causes the crash

## QGIS 3.16 (stand alone - Osgeo4W)

```Crash ID: 23e5bb055a4883e65250b0323e329dca40b4d88d
Stack Trace
QObject::isWidgetType :
QMainWindow::trUtf8 :
QHeaderView::logicalIndexAt :
QWidget::setStyleSheet :
QGIS Info
QGIS Version: 3.16.0-Hannover
QGIS code revision: 43b64b1
Compiled against Qt: 5.11.2
Running against Qt: 5.11.2
Compiled against GDAL: 3.1.4
Running against GDAL: 3.1.4
System Info
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.18362
```

Video 1
[![add_col_area_perimetro](https://img.youtube.com/vi/kdCmk_OHgrY/0.jpg)](https://youtu.be/kdCmk_OHgrY "QGIS 3.16 Hannover")

**Comment by pigreco** [comment]
I confirm

## OSGeo4W64b  Win 10 64b → clean profile

**Comment by roya0045** [comment]
Possibly related to the other properties #39720 #39693 and #39615  as it died on the same lines

**Comment by bellanz80** [comment]
> Possibly related to the other properties #39720 #39693 and #39615 as it died on the same lines

Thank you very much for noticing this interesting connection.

**Comment by roya0045** [comment]
@bellanz80 @pigreco what language is your QGIS in?

**Comment by pigreco** [comment]
> @bellanz80 @pigreco what language is your QGIS in?

Italy

**Comment by bellanz80** [comment]
> @bellanz80 @pigreco what language is your QGIS in?

English

**Comment by chleidersdorff** [comment]
I can confirm this happens on my machine as well.

QGIS Info
QGIS Version: 3.16.0-Hannover
QGIS code revision: 43b64b13f3
Compiled against Qt: 5.11.2
Running against Qt: 5.11.2
Compiled against GDAL: 3.1.4
Running against GDAL: 3.1.4



System Info
CPU Type: x86_64
Kernel Type: winnt
Kernel Version: 10.0.18363

**Comment by imanhn** [comment]
@luipir @nyalldawson 
I can confirm this fixes the crash.

Ran against Qt 5.11.2

**Comment by DelazJ** [comment]
Fixed in #39935

**Comment by luipir** [comment]
> Fixed in #39935

closing
